src/Symfony/Component/Validator/Constraints/CardSchemeValidator.php

```diff
@@ -79,7 +79,7 @@ class CardSchemeValidator extends ConstraintValidator
         ],
         // Payment system MIR numbers start with 220, then 1 digit from 0 to 4, then 12 digits
```

Comment:
This comment deserves an update too, doesn't it?

Reply:
```suggestion
        // Payment system MIR numbers start with 220, then 1 digit from 0 to 4, then 12 to 15 digits
```

Reply:
Done